sound/soc/sof/intel/hda-stream.c

```diff
@@ -433,15 +433,23 @@ int hda_dsp_stream_hw_params(struct snd_sof_dev *sdev,
 irqreturn_t hda_dsp_stream_interrupt(int irq, void *context)
 {
 	struct hdac_bus *bus = context;
+	struct sof_intel_hda_dev *sof_hda = bus_to_sof_hda(bus);
+	u32 stream_mask;
 	u32 status;
 
+	dev_vdbg(bus->dev, "stream irq, INTSTS status: 0x%x\n",
+		 snd_hdac_chip_readl(bus, INTSTS));
```

Comment:
is this necessary? If you add a log, then better add a log of the wrong interrupt being discarded

Reply:
I hit and debug these issues several rounds, so I want to add a verbose log here for the developer,  print out the whole register here is good enough for the developer, it has several bits, the developer need to refer to spec to understand it fully.